notification-portlet-webcomponents/notification-modal/src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <NotificationModal debug notificationApiUrl="http://localhost:8080/sample-notifications.json" />
+  <notification-modal debug notification-api-url="./sample-notifications.json"></notification-modal>
```

Comment:
Without this change, it was working on some browsers but not all?

Reply:
It works in all browsers either way.
This just makes it look more like the web component syntax.